network-mux/demo/cardano-ping.hs

```diff
@@ -126,14 +153,21 @@ main = do
 
     msgQueue <- newEmptyTMVarIO
 
-    when (isNothing $ host options) $ do
-        putStrLn "Specify host/ip with '-h <hostname>'"
+    when (isNothing (host options) && isNothing (unixSock options) ) $ do
+        putStrLn "Specify host/ip with '-h <hostname>' or a unix socket with -u <file name>"
         exitWith (ExitFailure 1)
 
-    addresses <- Socket.getAddrInfo (Just hints) (host options) (Just $ port options)
+    (addresses, versions) <- case unixSock options of
```

Comment:
The lines below seems to be over 80 lines, could you reduce the indentation.  Just two spaces is enough or break it further like this:
```
(...) <-
  case ... of
    ...
```